packages/presentation/src/loader/LoaderQueries.tsx

```diff
@@ -430,10 +430,7 @@ export function turboChargeResultIfSourceMap<T = unknown>(
         return undefined
       }
       // If there's a displayed document, always prefer it
-      if (
-        liveDocument?._id &&
-        getPublishedId(liveDocument._id) === getPublishedId(sourceDocument._id)
-      ) {
+      if (liveDocument?._id === sourceDocument._id) {
```

Comment:
This seems to fix the mismatch we discussed, but I assume there was a a good reason for comparing as published IDs? Will this change break something else?